lib/matplotlib/tests/test_style.py

```diff
@@ -184,6 +184,8 @@ def test_deprecated_seaborn_styles():
     with pytest.warns(mpl._api.MatplotlibDeprecationWarning):
         mpl.style.use("seaborn-bright")
     assert mpl.rcParams == seaborn_bright
+    with pytest.warns(mpl._api.MatplotlibDeprecationWarning):
```

Comment:
Can you explain this?  You just tested this three lines above - do we understand why that test passed?

Reply:
That's `style.use`, this is `style.library`.

Reply:
Ah...

OTOH, do we expect users to access `style.library`?  Should this be private, or is it useful to expose?

Reply:
I don't know if we expect them to or not, but we already have the bug report from a user, so it _is_ being used. It is also documented, at least a little, in the module docstring. In any case, this is only the bug fix for now.